### ISSUE TITLE
Windows installer  breaks when WSL install was previously used

### ISSUE THREAD

**Issue by TangentFoxy** [ollama/ollama#7460]
### What is the issue?

After discovering that Ollama was not automatically updating, I ran the installer to upgrade it (per FAQ instructions). Now a 500 error is generated if I try to access it, and the only logs and config available don't show anything useful? Additionally, the `ollama` command is not available.

![image](https://github.com/user-attachments/assets/7d650c18-b468-42dc-97ba-8b7a2907c787)
No additional info provided..

### app.log
```
time=2024-11-01T01:44:12.158-06:00 level=INFO source=logging.go:50 msg="ollama app started"
time=2024-11-01T01:44:12.381-06:00 level=INFO source=store.go:96 msg="wrote store: C:\\Users\\username\\AppData\\Local\\Ollama\\config.json"
time=2024-11-01T01:44:12.395-06:00 level=INFO source=store.go:96 msg="wrote store: C:\\Users\\username\\AppData\\Local\\Ollama\\config.json"
time=2024-11-01T01:44:12.400-06:00 level=INFO source=server.go:176 msg="unable to connect to server"
time=2024-11-01T01:44:12.400-06:00 level=INFO source=server.go:135 msg="starting server..."
time=2024-11-01T01:44:12.715-06:00 level=INFO source=server.go:121 msg="started ollama server with pid 19228"
time=2024-11-01T01:44:12.715-06:00 level=INFO source=server.go:123 msg="ollama server logs C:\\Users\\username\\AppData\\Local\\Ollama\\server.log"
```

### config.json
```json
{"id":"833b6b29-adbe-422e-98ec-5c78824cc785","first-time-run":true}
```

### server.log
Is an empty file.

### OS

Windows

### GPU

Nvidia

### CPU

AMD

### Ollama version

latest - I cannot say the *exact* version because it did not work

**Comment by TangentFoxy** [comment]
It seems that since I last looked at this project, the "Windows" version changed from using docker and WSL to run things to being its own thing separately? This perhaps explains why I wasn't getting updates, but it doesn't explain why the installer failed - since I had nothing else running at the time of install, and my previous install is working on an outdated version now that I've restarted it.

Would having multiple setups on the same machine like this be a reason the installer would fail? What is the difference under the hood now from how it worked before?

**Comment by dhiltgen** [comment]
Switching from WSL to the Windows native desktop app might have some rough edges.  Please make sure to uninstall the WSL linux component, and since you seem to have gotten into a weird failure state, I'd suggest uninstalling the windows app as well, and then do a fresh install.  If that doesn't clear it up, let us know and we can further troubleshoot.

**Comment by TangentFoxy** [comment]
Considering I got the WSL setup updated and working again, I will not be trying those steps at this time. (I was hoping for a more specific path in where to find what went wrong.) I am certainly open to trying it in the future though - would it increase performance because I'm avoiding whatever overhead WSL introduces?

I assume it would be best to close this issue now and only revisit it if I have trouble while trying to follow those steps in the future. Thank you!

(Part of why I decided not to try uninstalling and reinstalling right now is because of how the installer just throws everything into random places *without warning* on `C:/`, although, it seems this was just patched in #6967? That makes me very happy to see, especially with how many closed issues exist complaining about that problem!)
